test/operations.py

```diff
@@ -0,0 +1,94 @@
+from http import HTTPStatus
+import requests
+import json
+import os
+
+
+class Operations:
```

Comment:
Generally speaking, you only need a class if you're storing state.  In this case, we're not storing state so we're just using it as a namespace, which is fine... but a bit confusing.  Recommend you just turn these into module-level functions for clarity.